src/Catalyst.Benchmark/Catalyst.Abstractions/CryptoBenchmark.cs

```diff
@@ -95,6 +96,8 @@ public bool Verify(ISignature signature, ReadOnlySpan<byte> message, ReadOnlySpa
                 return true;
             }
 
+            public bool BatchVerify(IList<ISignature> signatures, IList<byte[]> messages, ReadOnlySpan<byte> context) { throw new NotImplementedException(); }
```

Comment:
is this correct?

Reply:
There is actually another repo I made a while ago with benchmarks for our crypto, including batch verification. I didn't know this project here existed until doing this bit of work. I'll make a ticket for consolidating the benchmarking into one place after testnet.

Reply:
I've created #1199 to fix this